pkg/users/local.js

```diff
@@ -734,18 +734,26 @@ PageAccount.prototype = {
            });
     },
 
-    get_locked: function() {
+    get_locked: function(update_display) {
+        update_display = typeof update_display !== 'undefined' ? update_display : true;
```

Comment:
This makes the default action of get_locked to call update - which happens in the code.
Only in the specific case that we need to try setting the value again do we care about setting this to false, that's why there's a default value.